code/modules/mob/holder.dm

```diff
@@ -49,7 +49,8 @@
 
 	var/obj/item/holder/mob_holder = new holder_type(loc)
 	src.forceMove(mob_holder)
-	mob_holder.name = loc.name
+	mob_holder.name = src.name
+	mob_holder.desc = src.desc
```

Comment:
```suggestion
	mob_holder.name = name
	mob_holder.desc = desc
```
You might also want to copy over the gender: https://www.byond.com/docs/ref/#/atom/var/gender
e.g. `mob_holder.gender = gender`